packages/contracts-bedrock/snapshots/storageLayout/L2ERC721Bridge.json

```diff
@@ -14,10 +14,31 @@
     "type": "bool"
   },
   {
-    "bytes": "1536",
-    "label": "__gap",
+    "bytes": "20",
+    "label": "spacer_0_2_20",
```

Comment:
@mds1 do you think this should be a `bytes30`?

Reply:
I agree with that. From re-reading [Storage Layout Management](https://www.notion.so/oplabs/Storage-Layout-Management-9fd6aa22b3cb4702b32fe65ae77ff21d?pvs=4#9b983922d0404316883968a09a3154dd) I don't think we ever came to consensus on a spacer format, but my suggestion would be to always use `bytesN` where `N` is the number of bytes needed to fill the rest of the slot without any gap. 

This is because if we later added a `bool` after this spacer, it would get packed into slot 0, and we're relying on the size of `messenger` to avoid packing.